Tests/kaas/k8s-default-storage-class/k8s-default-storage-class-check.py

```diff
@@ -180,28 +184,139 @@ def check_default_persistentvolumeclaim_readwriteonce(k8s_api_instance, storage_
                     "access mode 'ReadWriteOnce' is not supported",
                     return_code=42,
                 )
+    return 0
```

Comment:
What if the persistant volume is not found at all? Shouldn't this trigger an error??